integration-test/src/step_definitions/pdf_engine_client.js

```diff
@@ -3,17 +3,14 @@ const FormData = require('form-data');
 
 const pdf_engine_uri = process.env.PDF_ENGINE_URI;
 
-function generatePDF(zipFile, inputData, engineType) {
+function generatePDF(zipFile, inputData) {
```

Comment:
upd documentation to remove useless param https://pagopa.atlassian.net/wiki/spaces/PPR/pages/718012499/Design+Review+PDF+Engine#Request